deployments/Dockerfiles/tendermint/install.sh

```diff
@@ -0,0 +1,12 @@
+# Install architecture specific version of tendermint
+
+if [[ "$(uname -m)" == "amd64" ]]
+then
+    curl -L https://github.com/tendermint/tendermint/releases/download/v0.34.19/tendermint_0.34.19_linux_amd64.tar.gz > tendermint.tar.gz
+else
```

Comment:
why not specifically check for arm? and error out on else?

Reply:
I see, that would be futile since we only setup for building images on arm and amd platforms

Reply:
the current check assumes that if its not amd64 its arm64